### ISSUE TITLE
I18n translation isn't working for datepicker.

### ISSUE THREAD

**Issue by m-pereira** [TrestleAdmin/trestle#275]
Even setting: `config.i18n.default_locale = 'pt-BR'`, the datepicker doesn't work properly.

The format should be: `'d-m-Y'`, and the month name should be 'Janeiro'.

![trestle_issue1](https://user-images.githubusercontent.com/47258878/72933558-b6112680-3d40-11ea-920c-14a9e3b86dec.png)

Then I tried:
```
config.i18n.available_locales = ['pt-BR']
config.i18n.default_locale = 'pt-BR'
```

And the result was:

![trestle_issue2](https://user-images.githubusercontent.com/47258878/72933777-2ae46080-3d41-11ea-9eed-62a2686158c7.png)

**Comment by spohlenz** [comment]
Thank you for reporting. This was a couple of issues combined that should both now be fixed:

* Fixed incorrect datepicker format strings when the `en` locale is not available (5a5b6dc523286964f9582a89d675091b499d35d8)
* Fixed loading of Flatpickr translations for hyphenated locales like 'pt-BR' (11e5a7a7ab7fde6c3b90d306cdcde0fa88f28743)

If the default format strings are not appropriate for your locale, could you please open a PR that sets the locale strings for `admin.datepicker.formats.*`. See [here](https://github.com/TrestleAdmin/trestle/blob/master/config/locales/de.yml#L97-L101) for an example and [here](https://flatpickr.js.org/formatting/) for the Flatpickr docs on date format strings.

**Comment by m-pereira** [comment]
Thanks! That helped a lot! :clap: :clap:
